prime-router/src/main/kotlin/cli/tests/AuthTests.kt

```diff
@@ -729,7 +729,7 @@ class Server2ServerAuthTests : CoolTest() {
         val kid = "submission-testing-kid"
 
         if (environment == Environment.PROD) error("Can't create simple_report test data in PROD")
```

Comment:
Does this error message make sense now that the org is changed below from `simple_report` to `waters`?